codec-http2/src/main/java/io/netty/handler/codec/http2/Http2OutboundFrameLogger.java

```diff
@@ -121,6 +121,12 @@ public ChannelFuture writeWindowUpdate(ChannelHandlerContext ctx,
     @Override
     public ChannelFuture writeFrame(ChannelHandlerContext ctx, byte frameType, int streamId,
             Http2Flags flags, ByteBuf payload, ChannelPromise promise) {
+        return writer.writeFrame(ctx, frameType, streamId, flags.value(), payload, promise);
+    }
```

Comment:
It looks like we're missing the `logger.logUnknownFrame` that previously would have been called.
```suggestion
    public ChannelFuture writeFrame(ChannelHandlerContext ctx, byte frameType, int streamId,
            Http2Flags flags, ByteBuf payload, ChannelPromise promise) {
        return writeFrame(ctx, frameType, streamId, flags.value(), payload, promise);
    }
```

Reply:
thanks, that's right, fixed now.